### PR TITLE
update g4g notebook

### PR DESCRIPTION
**Pull request by biplovbhandari**
1. the licensing info  on the notebook,
2. added the link to the original repo (should already be there while cloning),
3. updated the clone to the main branch as you have,
4. removed the Make call to the Vertex AI endpoint via Earth Engine for now since that thing was not working, and
5. Updated the `Run in Colab` and `View on GitHub` in the notebook to the IMPACT’s master branch.

### CODE REVIEW

File: notebooks/IMPACT_SERVIR_HLS_FM_Customer_Container_G4G.ipynb [hunk 1]
```diff
@@ -29,22 +29,31 @@
         "id": "JAPoU8Sm5E6e"
       },
       "source": [
-        "# Deploying Harmonized Landsat and Sentinel-2 (HLS) Foundational Model to Vertex AI using custom container serving\n",
+        "# Deploying NASA-IMPACT HLS Based Foundational Model using FastAPI and Vertex AI custom container serving\n",
```
Comment (xhagrg) [inline notebooks/IMPACT_SERVIR_HLS_FM_Customer_Container_G4G.ipynb]: Would you be able to update this to "Deploying Harmonized Landsat and Sentinel-2 (HLS) Foundational Model to Vertex AI using custom container serving"

File: notebooks/IMPACT_SERVIR_HLS_FM_Customer_Container_G4G.ipynb [hunk 2]
```diff
@@ -53,7 +62,7 @@
       "source": [
         "## Overview\n",
         "\n",
-        "This tutorial walks you through building a custom container to serve finetuned HLS models on Vertex AI. We will be using FastAPI framework to create a prediction and health endpoint, following the conventions required by Vertex AI for custom model deployments. Google Cloud Platform (GCP) code build will be used to build the image used by vertex AI model deployment.\n",
+        "This tutorial walks you through building a custom container to serve any Machine Learning model on Vertex AI; here we will be using NASA-IMPACT's Harmonized Landsat Sentinel (HLS) based foundational model. We will be using FastAPI Python web server framework to create a prediction and health endpoint.\n",
```
Comment (xhagrg) [inline notebooks/IMPACT_SERVIR_HLS_FM_Customer_Container_G4G.ipynb]: Update this to "This tutorial walks you through building a custom container to serve fine-tuned HLS models on Vertex AI. We will be using FastAPI framework to create a prediction and health endpoint, following the conventions required by Vertex AI for custom model deployments. Google Cloud Platform (GCP) code build will be used to build the image used by vertex AI model deployment."

File: notebooks/IMPACT_SERVIR_HLS_FM_Customer_Container_G4G.ipynb [hunk 3]
```diff
@@ -78,10 +87,10 @@
...
         "The steps performed include:\n",
         "\n",
-        "- FastAPI server to handle predictions and health checks.\n",
-        "- Push GCP cloud build to build and push artifacts to GCP Artifcat Registry.\n",
-        "- Create Vertex AI Model from the custom image in Artifact Registry.\n",
-        "- Deploy the model to Vertex AI Endpoint for online prediction. "
+        "- Get the custom code ready with Docker container to be used in the Google's cloud run.\n",
+        "- Build a FastAPI server to handle predictions and health checks.\n",
+        "- Build a custom container with model artifacts.\n",
+        "- Upload and deploy custom container to Vertex AI Endpoints."
```
Comment (xhagrg) [inline notebooks/IMPACT_SERVIR_HLS_FM_Customer_Container_G4G.ipynb]: Use the following list:

"- FastAPI server to handle predictions and health checks.\n",
"- Push GCP cloud build to build and push artifacts to GCP Artifcat Registry.\n",
"- Create Vertex AI Model from the custom image in Artifact Registry.\n",
"- Deploy the model to Vertex AI Endpoint for online prediction. "

**Comment by biplovbhandari** [comment]
@xhagrg made the changes. It should be ready to merge now?

**Pull request merged**